KatJHuang/SpeechSentiments
# #1 create recorder api

This PR pertains to an atomic implementation of the recorder class. i.e. Recording is now a standalone functionality in the app. 

To test:
- Check this commit out with `git checkout <commit num>`
- Run app in emulator (Nexus 5X at API level 28)
- Do a few recording and playing and confirm there is no crash.
